src/vl-tooltip.js

```diff
@@ -455,6 +471,45 @@
   }
 
   /**
+  * Combine multiple binned fields in itemData into one field. The value of the field
+  * is a string that describes the bin range.
+  *
+  * @param {d3.map} itemData - a map of item.datum
+  * @param {Object[]} fieldOptions - a list of field options (i.e. options.fields[])
+  * @return itemData with combined bin fields
+  */
+  function combineBinFields(itemData, fieldOptions) {
+    if (!fieldOptions) return;
+
+    fieldOptions.forEach(function(fieldOption) {
+      if (fieldOption.bin === true) {
+
+        // get binned field names
+        var bin_field_start = fieldOption.field;
```

Comment:
Earlier in the code I was using `vl.fieldDef.field(fieldDef)` to get the full name of fields (i.e., with the prefixes and suffixes). For bin fields `vl.fieldDef.field()` gives me `bin_field_start`. I have added a line of code to change name from `bin_field_start` to `bin_field_range`. Range makes sense. So I think we are good now.